Makefile

```diff
@@ -221,10 +221,12 @@ $(PSALM): Makefile
 
 $(INFECTION): vendor $(shell find bin/ src/ -type f) $(BOX) box.json.dist .git/HEAD
 	composer require infection/codeception-adapter infection/phpspec-adapter
+	composer install --no-dev
```

Comment:
out of curiosity: I thought it was done automatically by Box. Isn't it the case anymore?

Reply:
It is, unfortunately https://github.com/box-project/box/issues/580 is still a problem; So even if Box removes the dev deps, at the moment, the dev plugins are still enabled hence Composer will try to load them (which will fail since the files are no longer here)

Reply:
Will add a note about this